packages/web-components/src/anchor-button/anchor-button.ts

```diff
@@ -259,6 +297,15 @@ export class AnchorButton extends BaseAnchor {
    */
   @attr({ attribute: 'icon-only', mode: 'boolean' })
   public iconOnly: boolean = false;
+
+  /**
+   * Handles changes to icon only custom states
+   * @param prev - the previous state
+   * @param next - the next state
+   */
+  public iconOnlyChanged(prev: boolean, next: boolean) {
+    toggleState(this.elementInternals, 'iconOnly', next);
```

Comment:
Just checking we want this to be 'iconOnly' vs 'icon-only'

Reply:
I'm open here - we may want it to be `icon` instead just for cleanliness...

Reply:
I went ahead and made this change - `iconOnly` => `icon`